apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/ApolloConfigChangeListener.java

```diff
@@ -74,4 +74,9 @@
    * If neither of {@code interestedKeys} and {@code interestedKeyPrefixes} is specified then the {@code listener} will be notified when whatever key is changed.
    */
   String[] interestedKeyPrefixes() default {};
+
+  /**
+   * Comma separated Apollo namespace for the config, merged with the ones in {@code value}
+   */
+  String commaSeparatedNamespaces() default "";
```

Comment:
How about we allow users to config comma-separated namespaces for the `value` field?

```java
  @ApolloConfigChangeListener(value = "${config.listeners}")
  public void onChange(ConfigChangeEvent changeEvent) {

  }
```

```properties
config.listeners = application,TEST1.apollo
```

Of course, we need to also update the documentation and demo usage in the [ConfigChangeListener.java](https://github.com/apolloconfig/apollo-java/blob/main/apollo-client/src/main/java/com/ctrip/framework/apollo/ConfigChangeListener.java).

Reply:
Hi @nobodyiam . Thanks for looking into this PR and giving your feedback. This sounds good, just had some thoughts on this

1. As per above suggestion does it mean, we _implicitly_ check the values to see if they are comma separated values ?
2. Although rare, in case some one has a legit comma (,) in their namespace identifier, this might cause an issue (unless namespaces naming has strict rules)
3. Just another thought - I think I made this configuration too specific for a given delimiter, what if someone uses another delimiter ?  I was thinking of something like this which can be both backward compatible to existing api and also provide flexibility.

```java
@ApolloConfigChangeListener(value = "${config.listeners}", delimiter = ",")
  public void onChange(ConfigChangeEvent changeEvent) {

  }
```

```
config.listeners = application,TEST1.apollo
```

`delimiter` would be an optional parameter (default value "") that can be added by the user when they wish to read multiple namespace from a single expression.

Please let me know your thoughts. Appreciate your feedback 🙇

Reply:
> 1. As per above suggestion does it mean, we implicitly check the values to see if they are comma separated values ?

Yes

> 2. Although rare, in case some one has a legit comma (,) in their namespace identifier, this might cause an issue (unless namespaces naming has strict rules)

`,` is not allowed in namespace name - Only digits, alphabets and symbol - _ . (except single .) are allowed
https://github.com/apolloconfig/apollo/blob/master/apollo-common/src/main/java/com/ctrip/framework/apollo/common/utils/InputValidator.java

> 3. Just another thought - I think I made this configuration too specific for a given delimiter, what if someone uses another delimiter ? I was thinking of something like this which can be both backward compatible to existing api and also provide flexibility.

This is a good point. But I think we could keep it simple and easy to understand to only support `,` as delimiter for now and see if there is real cases that would use other characters as the delimiter.

Reply:
Thanks for your feedback. It makes sense. Updated PR as per above suggestions.